### ISSUE TITLE
Uncaught SyntaxError: Unexpected token '<' | after "yarn add xlsx"

### ISSUE THREAD

**Issue by sharpapp-mareklanduch** [yarnpkg/yarn#7980]
### Bug description
Hi

I'm using yarn so I run "yarn add xlsx". I didn't do any imports just yarn add xlsx and I've got this error.
![image](https://user-images.githubusercontent.com/5566853/76295755-759c4680-62b5-11ea-9d67-f05105166f50.png)

Page cannot load.
Compilation is successful
![image](https://user-images.githubusercontent.com/5566853/76295783-82b93580-62b5-11ea-8dc0-50a8ef82209f.png)

I found out that when I'll do:
npm install xlsx
yarn install

then everything is working just fine, so I developed something simple and deployed my project on linux docker without "npm install xlsx" and it's also working there.
I am suspicious that maybe something wrong with the file system or fs library?

Tried clear cache, deleting node_modules, deleting yarn.lock and all those basic stuff :)

EDIT:
**NEW INFORMATION**
It turns out that when I will do such scenario
yarn install
yarn add xlsx
everything works

but if I'll do 
yarn install
yarn add xlsx
yarn install
error appears!

So after running yarn install some dependencies are installed and that's why it's failing :/
It's not good because when I'll commit my changes then other developers will install dependencies using "yarn install"

**Command**

```sh
yarn add xlsx
```

**What is the current behavior?**
I'm unable to work with some packages on windows machine using only yarn add command

**What is the expected behavior?**
I want be able to work with different packages including xlsx on windows machine. Without npm.

**Steps to Reproduce**

1. yarn add xlsx
2. yarn install
3. run project
4. white screen with following errors
![image](https://user-images.githubusercontent.com/5566853/76295755-759c4680-62b5-11ea-9d67-f05105166f50.png)

**Environment**

- Node Version: `12.13.1` - tested on various versions
- Yarn v1 Version: `1.21.1` 
- OS and version: Windows 10 v.1809

**Comment by sharpapp-mareklanduch** [comment]
I found that xlsx is working on 3.2.0 react scripts  version so it's probably not yarn fault

**Comment by sharpapp-mareklanduch** [comment]
https://github.com/facebook/create-react-app/issues/8490 - this thread helped me to solve my issue

I had unexpected token because I used PUBLIC_PATH variable in .env file
PUBLIC_PATH:/my-app/
I didn't have this variable in .env.development file.
So in react-scripts 3.2.0 it was by default "/" and after I updated react scripts it was failing with unexpected token error. I simply added PUBLIC_PATH:/ to my .env.development file.
